softdevice_controller/CHANGELOG.rst

```diff
@@ -9,10 +9,10 @@ Changelog
 
 All the notable changes to this project are documented in this file.
 
-nRF Connect SDK v2.1.0
-**********************
```

Comment:
I assume you need to do some changes here

Reply:
do you mean remove the entry completely?